generator/src/main/java/com/linkedin/pegasus/generator/FileFormatDataSchemaParser.java

```diff
@@ -48,7 +48,8 @@
  * @author Keren Jin
  * @author Joe Betz
  */
-public class FileFormatDataSchemaParser {
+public class FileFormatDataSchemaParser
+{
   static final String SCHEMA_PATH_PREFIX = SchemaDirectoryName.PEGASUS.getName() + "/";
   static final String EXTENSION_PATH_ENTRY = SchemaDirectoryName.EXTENSIONS.getName() + "/";
   private final String _resolverPath;
```

Comment:
The _resolverPath is not used anywhere, but It's an input of this class's constructor. I am thinking about shall we create a new constructor which remove the resolverPath as input. We could deprecate the current constructor later. But it is totally out of the scope of this PR, I just came to know it by reading this PR.

Reply:
Interesting... I didn't notice that 🤔  I can make the new constructor.

Reply:
Nevermind, deprecations should be done in minor releases. I'll deprecate it in the other PR.